libraries/security_cmd.rb

```diff
@@ -20,16 +20,28 @@ def add_certificates
       @keychain == '' ? [@security_cmd, 'add-certificates', @cert] : [@security_cmd, 'add-certificates', @cert, '-k', @keychain]
     end
 
-    def import(cert_passwd)
-      @keychain == '' ? [@security_cmd, 'import', @cert, '-P', cert_passwd] : [@security_cmd, 'import', @cert, '-P', cert_passwd, '-k', @keychain]
+    def import(cert_passwd, apps)
+      app_array = []
+
+      apps.each do |app|
+        app_array.push('-T')
+        app_array.push(app)
+      end
+      @keychain == '' ? [@security_cmd, 'import', @cert, '-P', cert_passwd, *app_array] : [@security_cmd, 'import', @cert, '-P', cert_passwd, '-k', @keychain, *app_array]
     end
 
-    def install_certificate(cert_passwd)
+    def install_certificate(cert_passwd, apps)
       valid_pkcs12 = ['.p12', '.pfx']
       valid_certs = ['.der', '.crt', '.cer']
 
+      apps.each do |app|
+        unless app.is_a? String
+          Chef::Exception.fatal("Invalid application: #{@app}.")
```

Comment:
Would anyone ever see this exception since the `property` method provides the type exception?